<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaInfo</class>
 <widget class="QWidget" name="MediaInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAlbumArtist">
       <property name="text">
        <string>Artist</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelAlbumTitle">
       <property name="text">
        <string>Album</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTrackNumber">
       <property name="text">
        <string>Track</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelAudioCodec">
       <property name="text">
        <string>Codec</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelAudioBitRate">
       <property name="text">
        <string>BitRate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="LightCycle" name="widgetLightCycle" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LightCycle</class>
   <extends>QWidget</extends>
   <header>lightcycle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
